contracts/Registry.vy

```diff
@@ -128,6 +133,16 @@ def _registerRelease(vault: address):
     log NewRelease(release_id, vault, Vault(vault).apiVersion())
 
 
+@internal
+def _registerToken(token: address):
```

Comment:
I considered adding a check here to make sure the token that's being added maps to a deployed vault, but since this check is already done in _registerDeployment it seemed like this check would be redundant and perhaps a waste of gas.

As long as _registerToken is only called from _registerDeployment (this is currently the case) this should not be a problem.

Alternatively, we can remove the internal method and move lines 138-143 to the bottom of _registerDeployment.

We could also keep this commit how it is as I believe the current version is more readable than the proposed alternative.

Reply:
Moved _registerToken logic into _registerDeployment